webapp/src/main/webapp/src/app/components/edit-labels/edit-labels.component.ts

```diff
@@ -16,46 +16,46 @@
 import { Component, EventEmitter, OnInit, Input, Output } from '@angular/core';
 
 @Component({
-  selector: 'edit-labels',
+  selector: 'app-edit-labels',
   templateUrl: './edit-labels.component.html',
   styleUrls: ['./edit-labels.component.css']
 })
 export class EditLabelsComponent implements OnInit {
-  @Output() onSave = new EventEmitter<Map<string, string>>();
+  @Output() save = new EventEmitter<Map<string, string>>();
 
-  @Input('resourceType') resourceType: string;
-  @Input('resourceName') resourceName: string;
-  @Input('labels') labels: any;
+  @Input() resourceType: string;
+  @Input() resourceName: string;
+  @Input() labels: any;
```

Comment:
Are you sure that removing the name of Input is safe here? Have you tested it in production mode? (with TS compilation and minification)

Reply:
https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin/docs/rules/no-input-rename.md#-invalid-code-5

Reply:
Thanks for references.